### PR TITLE
Gh 255 remove gaas common

### PR DESCRIPTION
**Pull request by rocky341**
Hi @t92549 

Small PR which removes GaaS-Common and re-integrates its modules back into GaaS-Rest as common is no longer necessary following the collapsing of the controller into rest.

#255

### CODE REVIEW

File: gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/model/Gaffer.java [hunk 1]
```diff
@@ -14,15 +14,16 @@
  * limitations under the License.
  */
 
-package uk.gov.gchq.gaffer.common.model.v1;
```
Comment (t92549) [inline gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/model/Gaffer.java:17]: Why is v1 being removed?

Comment (rocky341) [reply gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/model/Gaffer.java]: Sorry error while refactoring. Resolved.

**Review by t92549 — APPROVED** [review]
Cheers, ready to merge whenever

**Comment by go159lab** [comment]
> 

@t92549 Thank you for reviewing and updting the changes. Please merge this branch with upstream master

**Pull request merged**